src/prefect/blocks/notifications.py

```diff
@@ -486,3 +489,106 @@ def block_initialization(self) -> None:
+    )
+
+    chat_id: str = Field(
+        description=(
+            "Identify the users you want your bot to deliver your notifications to. You"
+            " must specify at least 1 chat_id. If you do not specify a chat_id, the"
+            " notification script will attempt to detect the bot owner's (you) chat_id"
+            " and use that."
+        ),
+    )
+
+    include_image: bool = Field(
+        default=False,
+        description=(
+            "You can optionally append the argument of ?image=Yes to the end of your"
+            " URL to have a Telegram message generated before the actual notice which"
+            " uploads the image associated with it. Due to the services limitations,"
+            " Telegram doesn't allow you to post an image inline with a text message."
+            " But you can send a message that just contains an image. If this flag is"
+            " set to true, apprise will send an image notification followed by the"
+            " notice itself. Since receiving 2 messages for every 1 notice could be"
+            " annoying to some, this has been made an option that defaults to being"
+            " disabled."
+        ),
+    )
+
+    silent: Optional[Literal["yes", "no"]] = Field(
+        default=None,
+        description=(
+            "A yes/no flag allowing you to send the notification in a silent fashion."
+            " By default this is set to no."
+        ),
+    )
+
+    preview: Optional[Literal["yes", "no"]] = Field(
+        default=None,
+        description=(
+            "A yes/no flag allowing you to display webpage previews of your post. By"
+            " default this is set to no."
+        ),
+    )
+
+    topic_id: Optional[SecretStr] = Field(
+        default=None,
+        description="The thread topic ID.",
+    )
+
+    detect_owner: bool = Field(
+        default=True,
+        description=(
+            "If detect_owner is set to True, we will attempt to determine who the bot"
+            " owner is based on the first person who messaged it. This is not a fool"
+            " proof way of doing things as over time Telegram removes the message"
+            " history for the bot.  So what appears (later on) to be the first message"
+            " to it, maybe another user who sent it a message much later.  Users who"
+            " set this flag should update their Apprise URL later to directly include"
+            " the user that we should message."
+        ),
+    )
+
+    def block_initialization(self) -> None:
+        from apprise.plugins.NotifyTelegram import NotifyTelegram
+
+        url = SecretStr(
+            NotifyTelegram(
+                bot_token=self.bot_token.get_secret_value(),
+                targets=self.chat_id,
+                include_image=self.include_image,
+                silent=self.silent,
+                preview=self.preview,
+                topic=self.topic_id,
+                detect_owner=self.detect_owner,
+            ).url()
```

Comment:
It'd be nice to include the `format` kwarg as well to allow different formatting styles.